docker-compose.yml

```
@@ -7,9 +7,12 @@ services:
     build:
       context: ./
       dockerfile: Dockerfile
-    command: ["/venv/bin/tracd", "--port", "9000", "-s", "trac-env"]
+    command: ["gunicorn", "--bind", "0:9000", "--reload", "tracdjangoplugin.wsgi"]
     environment:
      - TRAC_INI_database=postgres://code.djangoproject:secret@db/code.djangoproject
+     - TRAC_ENV=/code/trac-env
+     - SECRETS_FILE=/code/secrets.json
+     - DJANGO_SETTINGS_MODULE=tracdjangoplugin.settings
```

I get a `FileNotFoundError: [Errno 2] No such file or directory: '/code/secrets.json'` when running locally; perhaps the "Using Docker" section in the README could be updated with the necessary steps? (Possibly the non-Docker setup too, or remove that if we don't want to support it...)

Also, a nitpick: These look like the defaults and could be removed.

```suggestion
     - SECRETS_FILE=/code/secrets.json
```